daemon/src/editor.rs

```diff
@@ -108,12 +68,13 @@ pub async fn spawn_socket_listener(
     document_handle: DocumentActorHandle,
 ) -> Result<()> {
     // Make sure the parent directory of the socket is only accessible by the current user.
-    if let Err(description) = is_user_readable_only(&socket_path) {
+    /*if let Err(description) = is_user_readable_only(&socket_path) {
```

Comment:
You wrote in the PR:
> Maybe we should drop this permission requirement now? I think basically all other programs leave it up to the user to set permissions on files in their own directories like they want...

I have some "Bauchschmerzen" with this. We introduced this feature as part of an external security review. At least we should warn the user.

Though my thoughts go in the direction of owning the `.ethersync` directory: We are currently seeing at as flaw anyways that the user has to create `.ethersync` themselves, so maybe it's a good motivation to either create an `ethersync init` or to have a dialogue that asks the user if they want us to create `.ethersync`, if it doesn't exist when they call `ethersync share`.